### DISCUSSION TITLE
Cask brew outdated error - Error: No file found at https://raw.githubusercontent.com/Homebrew/homebrew-cask/3b6d4e07704f5953e29b96294d5ce58d1bfb9b02/Casks/blackhole-16ch.rb

### DISCUSSION THREAD

**Discussion by jimscard** [Homebrew/discussions#4200] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.21-89-ged79381
ORIGIN: https://github.com/Homebrew/brew
HEAD: ed793811553eb6fca3dd602931e37b7187f86a68
Last commit: 9 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 34b732b38991b8b2f26b592d87287ebc77937f19
Core tap last commit: 4 days ago
Core tap branch: master
Core tap JSON: 12 Feb 23:36 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.Of3BFWxZ3c/org.xquartz:0
HOMEBREW_EDITOR: /usr/local/bin/codium
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_UPDATE_REPORT_ALL_FORMULAE: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

>brew outdated
Error: No file found at https://raw.githubusercontent.com/Homebrew/homebrew-cask/3b6d4e07704f5953e29b96294d5ce58d1bfb9b02/Casks/blackhole-16ch.rb
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/api.rb:107:in `fetch_file_source'
/usr/local/Homebrew/Library/Homebrew/api/cask.rb:20:in `fetch_source'
/usr/local/Homebrew/Library/Homebrew/cask/cask_loader.rb:236:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/cask_loader.rb:376:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/caskroom.rb:50:in `block in casks'
/usr/local/Homebrew/Library/Homebrew/cask/caskroom.rb:46:in `map'
/usr/local/Homebrew/Library/Homebrew/cask/caskroom.rb:46:in `casks'
/usr/local/Homebrew/Library/Homebrew/cmd/outdated.rb:190:in `outdated_formulae_casks'
/usr/local/Homebrew/Library/Homebrew/cmd/outdated.rb:84:in `outdated'
/usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'

### Relevant casks

blackhole-16ch

**Comment by carlocab** [comment, marked as answer]
We're looking into it. Follow the discussion at https://github.com/Homebrew/homebrew-cask/issues/141436.

**Reply by Ender-Wang** [reply]
I just encountered this problem and coming to Git seeing your awareness 🤣
